src/controllers/event.js

```diff
@@ -1,16 +1,53 @@
-const createModal = require ( '../models/events.js');
-const {db} = require ('../databaseSetup');
-const {DataTypes} = require ('sequelize');
+const createModal = require("../models/events.js");
+const { db } = require("../databaseSetup");
+const { DataTypes } = require("sequelize");
 
 const Events = createModal(db, DataTypes);
 
 const getAllEvents = async (req, res) => {
-    try {
-        const events = await Events.findAll();
-        res.json(events);
-    } catch (error) {
-        res.json({ message: error.message });
-    }  
-}
+  const events = await Events.findAll();
+  res.json(events);
+};
+const getEventsById = async (req, res) => {
+  const events = await Events.findAll({
+    where: {
+      id: req.params.id,
+    },
+  });
+  res.json(events[0]);
+};
+const editEvents = async (req, res) => {
+  console.log(req.body);
+  await Events.update(req.body, {
+  where: {
+    id: req.params.id,
+  },
+});
+res.json({
+  message: 'UnitOfMeasurings Edited',
+});
+};
+const addEvents = async (req, res) => {
+  await Events.create(req.body, req.body);
```

Comment:
Why do two parameters in `create` method call are same?
Please remove second one